docs/fsharp/tutorials/getting-started/getting-started-visual-studio.md

```diff
@@ -5,168 +5,147 @@ keywords: visual f#, f#, functional programming
 author: dend
 manager: danielfe
 ms.date: 05/16/2016
-ms.topic: language-reference
+ms.topic: article
 ms.prod: visual-studio-dev14
 ms.technology: devlang-fsharp
 ms.assetid: 8db75596-19a9-4eda-b20d-a12d517c8cc1 
 ---
 
 # Getting Started with F# in Visual Studio
 
-> [!NOTE]
-This article is not up to date with the latest Visual Studio.  It will be updated.
+F# and the Visual F# tooling are supported in the Visual Studio IDE.  To begin, you should [download Visual Studio](https://visualstudio.com/downloads/).  The rest of the article will assume Visual Studio 2015 Community Edition.
 
-Visual Studio 2010 includes a new programming language, F#. F# is a multiparadigm language that supports functional programming in addition to traditional object-oriented programming and .NET concepts. The following examples introduce some of its features and syntax. The examples show how to declare simple variables, to write and test functions, to create tuples and lists, and to define and use a class.
+## Installing the Visual F# Tools
 
-Note: General Settings
+After Visual Studio is finished initializing, select **Custom** as shown here:
 
+![](../../media/vs2015-install-1.png)
```

Comment:
they need to be in a subfolder with the same name as the topic. so the reference should be:
./media/getting-started-visual-studio/vs2015-install-1.png